lib/file.js

```diff
@@ -57,7 +94,10 @@ File.prototype.setContent = function(content) {
  * @return {Object} Stats properties.
  */
 File.prototype.getStats = function() {
-  var size = this._content.length;
+  if (typeof this._size === 'undefined') {
+    this._size = this._content.length;
```

Comment:
I assume this will give unexpected results if `fs.stat()` is called before a read-related method (when `this._content` is still a function).